app/helpers/argo_helper.rb

```diff
@@ -102,11 +102,11 @@ def render_buttons(doc, object = nil)
     end
 
     # if this is an apo and the user has permission for the apo, let them edit it.
-    if (object.datastreams.include? 'roleMetadata') && (current_user.is_admin || current_user.is_manager || object.can_manage_item?(current_user.roles(apo_pid)))
+    if object.datastreams.include?('roleMetadata') && can?(:manage_item, object)
```

Comment:
My understanding of the following lines is that it only applies to objects that have a `roleMetadata` datastream. The comment suggests, though, an obvious refactor would be to check if the object is an APO.